instrumentation-genai/opentelemetry-instrumentation-vertexai-v2/tests/cassettes/test_vertexai_generate_content.yaml

```diff
@@ -0,0 +1,58 @@
+interactions:
+- request:
+    body: "{\n  \"contents\": [\n    {\n      \"role\": \"user\",\n      \"parts\":
```

Comment:
can you use the same formatter as openai, (yaml block scalar), as it makes this data a lot easier to read. Hopefully it is just pasting this https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/conftest.py#L118-L182

Reply:
Done thanks!